packages/trader/src/Stores/client-store.js

```diff
@@ -728,6 +728,7 @@ export default class ClientStore extends BaseStore {
                 amount  : obj_balance.total.real.amount,
                 currency: obj_balance.total.real.currency,
             };
+            this.resetLocalStorageValues(obj_balance.loginid);
```

Comment:
no, we don't want to set the login id that came in with balance as the active loginid. please send `this.loginid` instead. balance comes in for all the sibling accounts too